splinterd/sample_configs/splinterd.toml.example

```diff
@@ -18,35 +18,69 @@ version = "1"
 
 node_id = "012"
 
+display_name = "node-012"
```

Comment:
can you add descriptions for display name and node_id as well

Reply:
Yep, fixed!